azure-steps-windows.yml

```diff
@@ -27,7 +27,7 @@ steps:
   displayName: 'Download / Install OpenBLAS'
 
 - powershell: |
-    choco install -y mingw --forcex86 --force --version=5.3.0
+    choco install -y mingw --forcex86 --force --version=7.3.0
```

Comment:
In case it helps, SciPy is using `6.4.0` for 32-bit Windows. The most recent build seemed to be "ok" with it, but I suppose  NumPy may have different reqs?

Reply:
Oh, looking more closely, I think it is just a path problem. You may need to update to match the signature to what SciPy does (we repeat this for building and testing stages):

`$env:PATH = "C:\\ProgramData\\chocolatey\\lib\\mingw\\tools\\install\\mingw$(BITS)\\bin;" + $env:PATH`

Currently what NumPy has may use the wrong path: `$env:PATH = "C:\\tools\\mingw32\\bin;" + $env:PATH`

Reply:
thanks, adopted